spec/lib/search_spec.rb

```diff
@@ -3034,4 +3033,91 @@ def indexed_post(*args)
       expect(sql).to match(/where.*topics.closed/i)
     end
   end
+
+  describe "bot search" do
+    fab!(:bot) { Fabricate(:user, id: User.minimum(:id) - 1) }
```

Comment:
You can use `fab!(:bot)` here, we already have a bot fabricator:

https://github.com/discourse/discourse/blob/eca75c2ee9ee67f5de3e11edcc9dcbcb98e752e7/spec/fabricators/user_fabricator.rb#L145-L150

Reply:
oh did not realize